### ISSUE TITLE
[Site issue] No search results once word is complete

### ISSUE THREAD

**Issue by AndreKR** [blakadder/templates#1560]
When I enter "nou" into the search, I get the Nous devices, but when I enter "nous" I get no results:

https://user-images.githubusercontent.com/1188538/192843603-d3f636e6-44d5-4f1e-bf9b-45c8559ec7ac.mp4

**Comment by blakadder** [comment]
know issue, the js quasi search engine is like that

**Comment by AndreKR** [comment]
I took a quick look at this and I think the root cause is simply that `nous` gets stemmed into `nou` at index time, but at search time the added wildcard disables the stemming, and obviously a search for `nous` does not match an entry with `nou`.

There are several solutions to this but I think the best one is to disable stemming altogether because it is not really useful for names anyway:
```
self.pipeline.remove(lunr.stemmer)
self.searchPipeline.remove(lunr.stemmer)
```

While I was on it I noticed that in the case of multiple search terms the wildcard is only applied to the last one, so that `wall son` will retrieve Sonoff Wall Modules but `son wall` will not. I don't think this is on purpose, so I suggest we fix it. We can do that in the pipeline as well:
```
let leftAnchoredSearch = function (token) {
	token.update(function () { return token.toString() + '*' });
	return token;
};
lunr.Pipeline.registerFunction(leftAnchoredSearch, 'las');
self.searchPipeline.add(leftAnchoredSearch);
```

I have prepared PR #1561 with fixes for both issues.
